packages/roosterjs-react-emoji/lib/components/EmojiPane.tsx

```diff
@@ -38,13 +38,17 @@ export interface EmojiPaneProps {
     fullListClassName?: string;
     fullListContentClassName?: string;
     partialListClassName?: string;
-    onLayoutChange?: () => void;
+    navbarThemeClassName?: string;
```

Comment:
instead of doing this, expose the navbar's and statusbar's  props for override (use Partial<T> as the type, e.g. Partial<EmojiStatusBarProps>)

e.g. use of the prop after:
<EmojiStatusBar {...this.props.statusBarProps} emoji={this.getSelectedEmoji()} strings={strings} />